### PR TITLE
Update 5.9.0 Deployment Guide

### PR DESCRIPTION
**Pull request by YvonneW**
Renamed and updated the following files: 
- deployment-patterns.md
- fronting-with-the-nginx-load-balancer.md
- setting-up-separate-databases-for-clustering.md
- configuring-the-datasources.md
- deployment-guidelines-in-production.md

Renamed the following images and moved them to appropriate folders: 
- attachments/103329476/103329481.png ==> ../../assets/img/setup/deployment-with-the-loadbalancer.png
- attachments/103329499/103329500.png ==> ../../assets/img/setup/data-types.png
- attachments/56984556/61672631.png ==> ../../assets/img/setup/artifact-management.png
- attachments/103329463/103329465.png ==> ../../assets/img/setup/scenario.png
- attachments/103329463/103329464.png ==> ../../assets/img/setup/kestores.png
- attachments/103329466/103329467.png ==> ../../assets/img/setup/synchronous-token-persistence.png
- attachments/103329466/103329468.png ==> ../../assets/img/setup/asynchronous-token-persistence.png

### CODE REVIEW

File: en/mkdocs.yml [hunk 1]
```diff
@@ -78,19 +78,17 @@ nav:
...
+      - 'Deployment Patterns': 
         - 'Deployment Patterns': setup/deployment-patterns.md
-        - 'Setting Up Deployment Pattern 1': setup/setting-up-deployment-pattern-1.md
-        - 'Setting Up Deployment Pattern 2': setup/setting-Up-Deployment-Pattern-2.md
-        - 'Setting Up Separate Databases for Clustering': setup/setting-Up-Separate-Databases-for-Clustering.md
-      - 'Deployment Guidelines in Production': setup/deployment-Guidelines-in-Production.md
+        - 'Fronting with the Nginx load balancer': setup/fronting-with-the-nginx-load-balancer.md
+        - 'Configuring the Datasource' :setup/configuring-the-datasources.md
+        - 'Setting Up Separate Databases for Clustering': setup/setting-up-separate-databases-for-clustering.md
+      - 'Deployment Guidelines in Production': setup/deployment-suidelines-in-Production.md
```
Comment (sherenewso2) [inline en/mkdocs.yml]: "suidelines" should be corrected to "guidelines" in this folder path

Comment (sherenewso2) [inline en/mkdocs.yml]: "suidelines" should be changed to "guidelines" in this folder path

File: en/docs/setup/deployment-guidelines-in-production.md [hunk 2]
```diff
@@ -281,18 +247,15 @@ you can take the following approaches:
     sure to store sensitive data such as username and password to
     connect to the registry in a property file instead of in the Java
     code and secure the properties file with the [secure
-    vault](../../admin-guide/carbon-secure-vault-implementation)
+    vault](https://docs.wso2.com/display/ADMIN44x/Carbon+Secure+Vault+Implementation)
```
Comment (gomathyk) [inline en/docs/setup/deployment-guidelines-in-production.md]: Aren't we supposed to give the relative path here?

File: en/docs/setup/deployment-guidelines-in-production.md [hunk 3]
```diff
@@ -415,7 +373,7 @@ running the product.
 ### Hazelcast properties
 
 WSO2 products use
-[Hazelcast](https://docs.wso2.com/display/ADMIN44x/Clustering+Overview)
+[Hazelcast](../../admin-guide/clustering-Overview)
```
Comment (gomathyk) [inline en/docs/setup/deployment-guidelines-in-production.md]: 'O' in overview should be changed to simple case.

File: en/mkdocs.yml (hunk 1 above)
Comment (gomathyk) [inline en/mkdocs.yml]: correct the spelling of guidelines

Comment (YvonneW) [reply en/mkdocs.yml]: Fixed

File: en/docs/setup/deployment-guidelines-in-production.md (hunk 2 above)
Comment (YvonneW) [reply en/docs/setup/deployment-guidelines-in-production.md]: Fixed

File: en/docs/setup/deployment-guidelines-in-production.md (hunk 3 above)
Comment (YvonneW) [reply en/docs/setup/deployment-guidelines-in-production.md]: Fixed

File: en/mkdocs.yml (hunk 1 above)
Comment (YvonneW) [reply en/mkdocs.yml]: Fixed

File: en/docs/setup/deployment-guidelines-in-production.md (hunk 3 above)
Comment (gomathyk) [reply en/docs/setup/deployment-guidelines-in-production.md]: O in overview should be simple

**Pull request merged**